src/database/seeders/20201206072230-roommodel.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    
+      await queryInterface.bulkInsert('roomModels', [{
```

Comment:
This seed data will not be inserted, because the current table you have is roommodels, not roomModels, try running your seeds locally and see

Reply:
Yes, that is an error , I am going ahead and correct it